### ISSUE TITLE
Fails if .jvmopts is present with particular settings

### ISSUE THREAD

**Issue by tampler** [scalameta/metals#1496]
Ubuntu 19
OpenJDK 11 
VSCode 1.43.0
Metals 0.8.1

When I open a project, which has `.jvmopts` with any of commented settings with `#`
Metals fails immediately with the following message:
```
Failed to download Metals, make sure you have an internet connection and the Java Home '/usr/lib/jvm/java-11-openjdk-amd64' is valid. You can configure the Java Home in the settings
```
If I remove it, it works. My .jvmsettings file is:
```bash
-XX:+CMSClassUnloadingEnabled
-Xmx2G
-Xms2G
-XX:MaxMetaspaceSize=1g
-Xss8m 

#-XX:+UseNUMA 
#-XX:+UnlockExperimentalVMOptions 
#-XX:+UseZGC
#-XX:ConcGCThreads=2
```

**Comment by olafurpg** [comment]
Thank you for looking! I moved it back to Metals but issue still seems related to the VS Code issue since it's unable to start Java 🤔

**Comment by tampler** [comment]
Guys, I'll do more testing on this later today

**Comment by gabro** [comment]
Thanks! It may actually be that one of the options is responsible and it would be interesting to pinpoint which one

**Comment by tampler** [comment]
No, Gabro. I tried all four since I that was a refactor from `.sbtopts` in one of my commercial projects

How's the situation with Corona in Italy? Hope you are fine there...

**Comment by gabro** [comment]
Ok, let me know if you manage to find the root cause. I'm not sure when I'll have time to reproduce this.

For the Coronavirus situation, it's serious, we're basically locked in our homes hoping that it will relieve some pressure off our healthcare system. I'm personally fine, but it's a surreal situation!

**Comment by tampler** [comment]
@gabro Alright it seems like VSCode fails on the following pattern: `-XX:+U` 
Cos it fails on the last three options, which I commented 
But allows `-XX:+CMSClassUnloadingEnabled` in the beginning

**Comment by gabro** [comment]
@tampler thanks for checking. So can you confirm an offending `.jvmopts` would be for example:

```
-XX:+UseNUMA 
```

?

**Comment by gabro** [comment]
Also, which Java version are you using? This should be visible in `.metals/metals.log`

**Comment by tampler** [comment]
Use, -XX:+UseNUMA is the victim. See a screen attached 
openjdk version "11.0.6" 2020-01-14

![metals](https://user-images.githubusercontent.com/2816743/76613600-81933d00-652f-11ea-9f53-62822632c3aa.png)

**Comment by gabro** [comment]
Ok, I've tried to set `.jvmopts` to

```
-XX:+UseNUMA
```

on the Metals repo and reload the window, but I was not able to reproduce :(

I'm using adoptopenjdk 11.0.6

**Comment by tampler** [comment]
Strange... Are you working on Ubuntu ? Mine is Ubuntu 19 and VSCode 1.43.0

**Comment by gabro** [comment]
No, I'm on macOS. That might be a point of difference

**Comment by tgodzik** [comment]
Checked on Ubuntu and it seems the issue is:
```
Unrecognized VM option 'UseNUMA '
Did you mean '(+/-)UseNUMA'?
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

It's the space at the end, we don't trim the options.
